epoxy-processortest/build.gradle

```diff
@@ -16,7 +16,21 @@ android {
 dependencies {
   testCompile project(':epoxy-processor')
   testCompile project(':epoxy-processortest2')
-  testCompile files(Jvm.current().getRuntimeJar())
+  testCompile files(getRuntimeJar())
   testCompile files(Jvm.current().getToolsJar())
   testCompile rootProject.deps.googleTestingCompile
 }
+
+static def getRuntimeJar() {
+  try {
+    final File javaBase = new File(System.getProperty("java.home")).getCanonicalFile();
```

Comment:
wow, thanks for doing this. I tried to bump gradle but wasn't sure how to change this correctly. How did you know what to do btw?